### ISSUE TITLE
Invalid Format

### ISSUE THREAD

**Issue by SRNV** [kt3k/deno-bin#3]
I get this error after a `npm i --save-dev deno-bin`
```shell
> deno-bin@1.8.3-r install /home/rudy/.vscode/extensions/Otone/server/node_modules/deno-bin
> node install.js

/home/rudy/.vscode/extensions/Otone/server/node_modules/adm-zip/zipFile.js:101
                        throw new Error(Utils.Errors.INVALID_FORMAT);
                        ^

Error: Invalid or unsupported zip format. No END header found
    at readMainHeader (/home/rudy/.vscode/extensions/Otone/server/node_modules/adm-zip/zipFile.js:101:10)
    at new module.exports (/home/rudy/.vscode/extensions/Otone/server/node_modules/adm-zip/zipFile.js:17:9)
    at new module.exports (/home/rudy/.vscode/extensions/Otone/server/node_modules/adm-zip/adm-zip.js:61:18)
    at WriteStream.res.pipe.on (/home/rudy/.vscode/extensions/Otone/server/node_modules/deno-bin/install.js:44:7)
    at WriteStream.emit (events.js:198:13)
    at lazyFs.close (internal/fs/streams.js:207:14)
    at FSReqWrap.args [as oncomplete] (fs.js:140:20)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! deno-bin@1.8.3-r install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the deno-bin@1.8.3-r install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/rudy/.npm/_logs/2021-04-07T12_37_52_194Z-debug.log
```

**Comment by SRNV** [comment]
the version format seems to break the install
works fine if I specify a version in the command `npm i -save-dev deno-bin@1.8.2`
```shell
> deno-bin@1.8.2 install /home/rudy/.vscode/extensions/Otone/server/node_modules/deno-bin
> node install.js

+ deno-bin@1.8.2
added 3 packages from 5 contributors and audited 15 packages in 4.351s

5 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

**Comment by kt3k** [comment]
@SRNV Thank you for the report! The latest version had error in handling version number. I published `v1.8.3-r2` to npm, which should download deno v1.8.3.

**Comment by kt3k** [comment]
this should be solved now in main
